ui/apps/platform/src/Containers/MainPage/Sidebar/NavigationSidebar.tsx

```diff
@@ -32,6 +32,7 @@ import {
     vulnManagementRiskAcceptancePath,
     vulnerabilitiesWorkloadCvesPath,
     vulnerabilityReportsPath,
+    exceptionManagementPath,
```

Comment:
Not a blocker to merge, because I can move in follow up for separator, according to comment above:

```js
// Import path variables in alphabetical order to minimize merge conflicts when multiple people add routes.
```

Reply:
@pedrottimark let's move it in your PR